Resources/Prototypes/Nyanotrasen/Entities/Objects/Books/hyperlinks.yml

```diff
@@ -202,3 +226,19 @@
   - type: GuidebookBook
     guides:
     - Atmospherics
+
+- type: entity
+  parent: BaseHyperlinkBook
+  id: HyperlinkBookEngineering
+  suffix: GuidebookBook
+  name: The engineer's guide to the station
+  description: A guide on Engineering. A combination of various other engineering books and more!
+  components:
+  - type: Sprite
+    layers:
+    - state: book_engineering2
+  # - type: HyperlinkBook
+  #   url: https://wiki.nyanotrasen.moe/view/Atmospheric_Science
+  - type: GuidebookBook
+    guides:
+    - Engineering
```

Comment:
This should be in our guidebook file. Better yet, just replace the construction book with this, since that's a subpage of this one anyway.

```suggestion
```

EDIT- Actually, scratch that. It's a little more complicated. We already have Power, Atmos, and Construction books, but no Reverse Engineering, or Machine Upgrading, and the main Engineering page isn't anywhere.